# ==============================================================================
# REGRESSOR COMMIT DIFF
# ==============================================================================
# File: dom/webidl/VideoEncoder.webidl
# Commit: ea4d0db67f89
# Full Hash: ea4d0db67f899714106e6f4ffcc78ebe8feaf466
# Author: Paul Adenot <paul@paul.cx>
# Date: 2023-12-23 09:52:03
# Regressor Bug: 1749047
# File Overlap Count: 1
# Description:
#   Bug 1749047 - Implement plumbing for SVC. r=chunmin,webidl,smaug
#   
#   Differential Revision: https://phabricator.services.mozilla.com/D196473
# ==============================================================================

diff -r f0ac27321cf8 -r ea4d0db67f89 dom/webidl/VideoEncoder.webidl
--- a/dom/webidl/VideoEncoder.webidl	Fri Dec 22 21:39:18 2023 +0000
+++ b/dom/webidl/VideoEncoder.webidl	Fri Dec 22 21:39:18 2023 +0000
@@ -110,13 +110,11 @@
 
 dictionary EncodedVideoChunkMetadata {
   VideoDecoderConfig decoderConfig;
-  // Not implemented https://bugzilla.mozilla.org/show_bug.cgi?id=1867066
-  // SvcOutputMetadata svc;
+  SvcOutputMetadata svc;
   // Not implemented https://bugzilla.mozilla.org/show_bug.cgi?id=1867067
   // BufferSource alphaSideData;
 };
 
-// Not implemented https://bugzilla.mozilla.org/show_bug.cgi?id=1867066
-// dictionary SvcOutputMetadata {
-//   unsigned long temporalLayerId;
-// };
+dictionary SvcOutputMetadata {
+  unsigned long temporalLayerId;
+};
